pkg/cmd/opts/install.go

```diff
@@ -382,7 +382,8 @@ func (o *CommonOptions) GetLatestJXVersion(resolver *versionstream.VersionResolv
 		return util.GetLatestVersionStringFromBucketURLs(latestVersionStrings)
 	}
 
-	dir := resolver.VersionsDir
+	dir := resolver.GetVersionsDir()
+	fmt.Println("Gone past this")
```

Comment:
Pretty sure you meant to remove this. =)

Reply:
LOL :+1:

Reply:
Did a git commit and push from a different directory (pkg/cmd ..) instead of the base jx directory